mithril-client-cli/Cargo.toml

```diff
@@ -52,13 +48,10 @@ slog = { version = "2.7.0", features = [
 slog-async = "2.8.0"
 slog-scope = "4.4.0"
 slog-term = "2.9.0"
-tar = "0.4.40"
 thiserror = "1.0.49"
 tokio = { version = "1.32.0", features = ["full"] }
-zstd = "0.13.0"
 
 [dev-dependencies]
-httpmock = "0.6.8"
 mithril-common = { path = "../mithril-common", features = ["test_http_server"] }
 mockall = "0.11.4"
 warp = "0.3"
```

Comment:
Do we still need `warp`?

Reply:
Yes we still need `warp` to use `TestHttpServer` in `add_statistics` unit test (which will be removed in the next PR).